course_catalog/tasks.py

```diff
@@ -122,3 +125,87 @@ def get_youtube_data(*, channel_ids=None):
     results = pipelines.youtube_etl(channel_ids=channel_ids)
 
     return len(list(results))
+
+
+def get_youtube_videos_for_transcripts_job(
+    *, created_after=None, created_minutes=None, overwrite=False
+):
+    """
+    course_catalog.Video object filtered to get_youtube_transcripts job params
+    Args:
+        created_after (str or None):
+            if a string inclued only videos with a created_on after that date
+        created_minutes (str or None):
+            if a string include only videos with created_on in the last created_minutes minutes
+        overwrite (bool):
+            if true include videos that already have transcripts
+    Returns
+        Django filtered course_catalog.videos object
+    """
+
+    videos = Video.objects
+    if not overwrite:
+        videos = videos.filter(transcript="")
+
+    if created_after:
+        try:
+            date = datetime.strptime(created_after, ISOFORMAT).replace(tzinfo=pytz.UTC)
+        except ValueError:
+            log.exception("Invalid date format")
+            return
+        videos = videos.filter(created_on__gte=date)
+    elif created_minutes:
+        try:
+            created_minutes = int(created_minutes)
+        except ValueError:
+            log.exception("created_minutes must be an integer")
+            return
+
+        date = datetime.utcnow() - timedelta(minutes=created_minutes)
+        date.replace(tzinfo=pytz.UTC)
+        videos = videos.filter(created_on__gte=date)
+    return videos
+
+
+@app.task
+def get_youtube_transcripts(
```

Comment:
Task implementations should be fairly thin wrappers around APIs, can you refactor the body of this task into a `youtube.py` function?